### PR TITLE
Readme changes

### PR DESCRIPTION
**Pull request by Mohan29997**
Merge Request For read me file related Overview, Getting started, setup and Architecture

### CODE REVIEW

**Review by faiyyazs — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -106,6 +174,44 @@ The layers ```core``` and ```services provider``` within infrastructure each hav
...
   - Responsive Farmework
 
+  ## Run Projects
+### 1. install
+#### Install
+```shell
+$ yarn install
+```
+
+### 2. Mock Server
```
Comment (faiyyazs) [inline README.md:184]: We dont have a mock-server

File: README.md [hunk 2]
```diff
@@ -106,6 +174,44 @@ The layers ```core``` and ```services provider``` within infrastructure each hav
...
+#### Start
+```shell
+$ yarn run mock-server
+```
+
+### 3. Web
```
Comment (faiyyazs) [inline README.md:190]: All these commands will be specific to our workspace

File: wiki/DependecnyManagment.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Dependency Injection
+Dependency injection is necessary if you're not coding spaghetti 🍝 and you want to keep nice layers of separation in your React Native app's codebase. The problem is that all of the libraries out there, such as get_it or kiwi, are just service locators with no support or a limited support for automating the registration of dependencies.
+
+Dagger solves it elegantly for native Android and Angular is also known for its powerful dependency injection framework. Now, we React Native  developers can finally use something similar
+
+# Why use GetIt and Injectable?
```
Comment (faiyyazs) [inline wiki/DependecnyManagment.md:6]: We dont have GETIT & Injectable. Please remove these tools

File: wiki/DependecnyManagment.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Dependency Injection
+Dependency injection is necessary if you're not coding spaghetti 🍝 and you want to keep nice layers of separation in your React Native app's codebase. The problem is that all of the libraries out there, such as get_it or kiwi, are just service locators with no support or a limited support for automating the registration of dependencies.
+
+Dagger solves it elegantly for native Android and Angular is also known for its powerful dependency injection framework. Now, we React Native  developers can finally use something similar
```
Comment (faiyyazs) [inline wiki/DependecnyManagment.md:4]: Nor dagger. We will be using react native specific tools for DI

**Review by faiyyazs — COMMENTED** [review]
The Readme needs to be specific to react native

**Comment by Mohan29997** [comment]
remove mock server and the dependency injection part which was related to flutter

**Review by faiyyazs — APPROVED** [review]

**Pull request merged**